<!doctype html> <html> <head><style>
             table { border-collapse: collapse; }
             th, td {
               word-wrap: break-word;
               max-width: 100%;
               font-family: "Trebuchet MS", Arial, Helvetica, sans-serif;
               border-bottom: 1px solid #ddd;
               padding: 5px;
               text-align: left;
             }
            tr:hover {background: #f4f4f4;}
            tr:hover .highlighted {background: repeating-linear-gradient(
                    45deg,
                    #ffff99,
                    #ffff99 10px,
                    #f4f4f4 10px,
                    #f4f4f4 20px
                  );}
           .highlighted { background-color: #ffff99; }
          </style></head><body><h2>List of sequence alignment software</h2><br> <b>Section Title</b>: Pairwise alignment <br><b>Table Section Text</b>: *Sequence type: protein or nucleotide **Alignment type: local or global <br> <table>
<tr> <th colspan=1 rowspan=1 > Name </th><th colspan=1 rowspan=1 > Description </th><th colspan=1 rowspan=1 > Sequence type* </th><th colspan=1 rowspan=1 > Alignment type** </th><th colspan=1 rowspan=1 > Link </th><th colspan=1 rowspan=1 > Author </th><th colspan=1 rowspan=1 > Year </th></tr>
<tr> <td colspan=1 rowspan=1 > ACANA </td><td colspan=1 rowspan=1 > Fast heuristic anchor based pairwise alignment </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > download </td><td colspan=1 rowspan=1 > Huang, Umbach, Li </td><td colspan=1 rowspan=1 > 2005 </td></tr>
<tr> <td colspan=1 rowspan=1 > AlignMe </td><td colspan=1 rowspan=1 > Alignments for membrane protein sequences </td><td colspan=1 rowspan=1 > Protein </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > download, server </td><td colspan=1 rowspan=1 > M. Stamm, K. Khafizov, R. Staritzbichler, L.R. Forrest </td><td colspan=1 rowspan=1 > 2013 </td></tr>
<tr> <td colspan=1 rowspan=1 > ALLALIGN </td><td colspan=1 rowspan=1 > For DNA, RNA and proteins with summed length n, generates all local alignments in O(n) time using approximate suffix tree matching or mapped density dynamic alignment </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Local </td><td colspan=1 rowspan=1 > allalign </td><td colspan=1 rowspan=1 > E. Wachtel </td><td colspan=1 rowspan=1 > 2017 </td></tr>
<tr> <td colspan=1 rowspan=1 > Bioconductor Biostrings::pairwiseAlignment </td><td colspan=1 rowspan=1 > Dynamic programming </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Both + Ends-free </td><td colspan=1 rowspan=1 > site </td><td colspan=1 rowspan=1 > P. Aboyoun </td><td colspan=1 rowspan=1 > 2008 </td></tr>
<tr> <td colspan=1 rowspan=1 > BioPerl dpAlign </td><td colspan=1 rowspan=1 > Dynamic programming </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Both + Ends-free </td><td colspan=1 rowspan=1 > site </td><td colspan=1 rowspan=1 > Y. M. Chan </td><td colspan=1 rowspan=1 > 2003 </td></tr>
<tr> <td colspan=1 rowspan=1 > BLASTZ, LASTZ </td><td colspan=1 rowspan=1 > Seeded pattern-matching </td><td colspan=1 rowspan=1 > Nucleotide </td><td colspan=1 rowspan=1 > Local </td><td colspan=1 rowspan=1 > download, download </td><td colspan=1 rowspan=1 > Schwartz et al. </td><td colspan=1 rowspan=1 > 2004,2009 </td></tr>
<tr> <td colspan=1 rowspan=1 > CUDAlign </td><td colspan=1 rowspan=1 > DNA sequence alignment of unrestricted size in single or multiple GPUs </td><td colspan=1 rowspan=1 > Nucleotide </td><td colspan=1 rowspan=1 > Local, SemiGlobal, Global </td><td colspan=1 rowspan=1 > download </td><td colspan=1 rowspan=1 > E. Sandes </td><td colspan=1 rowspan=1 > 2011-2015 </td></tr>
<tr> <td colspan=1 rowspan=1 > DNADot </td><td colspan=1 rowspan=1 > Web-based dot-plot tool </td><td colspan=1 rowspan=1 > Nucleotide </td><td colspan=1 rowspan=1 > Global </td><td colspan=1 rowspan=1 > server </td><td colspan=1 rowspan=1 > R. Bowen </td><td colspan=1 rowspan=1 > 1998 </td></tr>
<tr> <td colspan=1 rowspan=1 > DNASTAR Lasergene Molecular Biology Suite </td><td colspan=1 rowspan=1 > Software to align DNA, RNA, protein, or DNA + protein sequences via pairwise and multiple sequence alignment algorithms including MUSCLE, Mauve, MAFFT, Clustal Omega, Jotun Hein, Wilbur-Lipman, Martinez Needleman-Wunsch, Lipman-Pearson and Dotplot analysis. </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > DNASTAR site </td><td colspan=1 rowspan=1 > DNASTAR </td><td colspan=1 rowspan=1 > 1993-2016 </td></tr>
<tr> <td colspan=1 rowspan=1 > DOTLET </td><td colspan=1 rowspan=1 > Java-based dot-plot tool </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Global </td><td colspan=1 rowspan=1 > applet </td><td colspan=1 rowspan=1 > M. Pagni and T. Junier </td><td colspan=1 rowspan=1 > 1998 </td></tr>
<tr> <td colspan=1 rowspan=1 > FEAST </td><td colspan=1 rowspan=1 > Posterior based local extension with descriptive evolution model </td><td colspan=1 rowspan=1 > Nucleotide </td><td colspan=1 rowspan=1 > Local </td><td colspan=1 rowspan=1 > site </td><td colspan=1 rowspan=1 > A. K. Hudek and D. G. Brown </td><td colspan=1 rowspan=1 > 2010 </td></tr>
<tr> <td colspan=1 rowspan=1 > Genome Compiler Genome Compiler </td><td colspan=1 rowspan=1 > Align chromatogram files (.ab1, .scf) against a template sequence, locate errors, and correct them instantly. Learn more </td><td colspan=1 rowspan=1 > Nucleotide </td><td colspan=1 rowspan=1 > Local </td><td colspan=1 rowspan=1 > Free online & download </td><td colspan=1 rowspan=1 > Genome Compiler Corporation </td><td colspan=1 rowspan=1 > 2014 </td></tr>
<tr> <td colspan=1 rowspan=1 > G-PAS </td><td colspan=1 rowspan=1 > GPU-based dynamic programming with backtracking </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Local, SemiGlobal, Global </td><td colspan=1 rowspan=1 > site+download </td><td colspan=1 rowspan=1 > W. Frohmberg, M. Kierzynka et al. </td><td colspan=1 rowspan=1 > 2011 </td></tr>
<tr> <td colspan=1 rowspan=1 > GapMis </td><td colspan=1 rowspan=1 > Does pairwise sequence alignment with one gap </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > SemiGlobal </td><td colspan=1 rowspan=1 > site </td><td colspan=1 rowspan=1 > K. Frousios, T. Flouri, C. S. Iliopoulos, K. Park, S. P. Pissis, G. Tischler </td><td colspan=1 rowspan=1 > 2012 </td></tr>
<tr> <td colspan=1 rowspan=1 > GGSEARCH, GLSEARCH </td><td colspan=1 rowspan=1 > Global:Global (GG), Global:Local (GL) alignment with statistics </td><td colspan=1 rowspan=1 > Protein </td><td colspan=1 rowspan=1 > Global in query </td><td colspan=1 rowspan=1 > FASTA server </td><td colspan=1 rowspan=1 > W. Pearson </td><td colspan=1 rowspan=1 > 2007 </td></tr>
<tr> <td colspan=1 rowspan=1 > JAligner </td><td colspan=1 rowspan=1 > Java open-source implementation of Smith-Waterman </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Local </td><td colspan=1 rowspan=1 > JWS </td><td colspan=1 rowspan=1 > A. Moustafa </td><td colspan=1 rowspan=1 > 2005 </td></tr>
<tr> <td colspan=1 rowspan=1 > K*Sync </td><td colspan=1 rowspan=1 > Protein sequence to structure alignment that includes secondary structure, structural conservation, structure-derived sequence profiles, and consensus alignment scores </td><td colspan=1 rowspan=1 > Protein </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Robetta server </td><td colspan=1 rowspan=1 > D. Chivian & D. Baker </td><td colspan=1 rowspan=1 > 2003 </td></tr>
<tr> <td colspan=1 rowspan=1 > LALIGN </td><td colspan=1 rowspan=1 > Multiple, non-overlapping, local similarity (same algorithm as SIM) </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Local non-overlapping </td><td colspan=1 rowspan=1 > server FASTA server </td><td colspan=1 rowspan=1 > W. Pearson </td><td colspan=1 rowspan=1 > 1991 (algorithm) </td></tr>
<tr> <td colspan=1 rowspan=1 > NW-align </td><td colspan=1 rowspan=1 > Standard Needleman-Wunsch dynamic programming algorithm </td><td colspan=1 rowspan=1 > Protein </td><td colspan=1 rowspan=1 > Global </td><td colspan=1 rowspan=1 > server and download </td><td colspan=1 rowspan=1 > Y Zhang </td><td colspan=1 rowspan=1 > 2012 </td></tr>
<tr> <td colspan=1 rowspan=1 > mAlign </td><td colspan=1 rowspan=1 > modelling alignment; models the information content of the sequences </td><td colspan=1 rowspan=1 > Nucleotide </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > doc code[permanent dead link] </td><td colspan=1 rowspan=1 > D. Powell, L. Allison and T. I. Dix </td><td colspan=1 rowspan=1 > 2004 </td></tr>
<tr> <td colspan=1 rowspan=1 > matcher </td><td colspan=1 rowspan=1 > Waterman-Eggert local alignment (based on LALIGN) </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Local </td><td colspan=1 rowspan=1 > Pasteur </td><td colspan=1 rowspan=1 > I. Longden (modified from W. Pearson) </td><td colspan=1 rowspan=1 > 1999 </td></tr>
<tr> <td colspan=1 rowspan=1 > MCALIGN2 </td><td colspan=1 rowspan=1 > explicit models of indel evolution </td><td colspan=1 rowspan=1 > DNA </td><td colspan=1 rowspan=1 > Global </td><td colspan=1 rowspan=1 > server </td><td colspan=1 rowspan=1 > J. Wang et al. </td><td colspan=1 rowspan=1 > 2006 </td></tr>
<tr> <td colspan=1 rowspan=1 > MUMmer </td><td colspan=1 rowspan=1 > suffix tree based </td><td colspan=1 rowspan=1 > Nucleotide </td><td colspan=1 rowspan=1 > Global </td><td colspan=1 rowspan=1 > download </td><td colspan=1 rowspan=1 > S. Kurtz et al. </td><td colspan=1 rowspan=1 > 2004 </td></tr>
<tr> <td colspan=1 rowspan=1 > needle </td><td colspan=1 rowspan=1 > Needleman-Wunsch dynamic programming </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > SemiGlobal </td><td colspan=1 rowspan=1 > EMBL-EBIPasteur </td><td colspan=1 rowspan=1 > A. Bleasby </td><td colspan=1 rowspan=1 > 1999 </td></tr>
<tr> <td colspan=1 rowspan=1 > Ngila </td><td colspan=1 rowspan=1 > logarithmic and affine gap costs and explicit models of indel evolution </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Global </td><td colspan=1 rowspan=1 > download </td><td colspan=1 rowspan=1 > R. Cartwright </td><td colspan=1 rowspan=1 > 2007 </td></tr>
<tr> <td colspan=1 rowspan=1 > NW </td><td colspan=1 rowspan=1 > Needleman-Wunsch dynamic programming </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Global </td><td colspan=1 rowspan=1 > download </td><td colspan=1 rowspan=1 > A.C.R. Martin </td><td colspan=1 rowspan=1 > 1990-2015 </td></tr>
<tr> <td colspan=1 rowspan=1 > parasail </td><td colspan=1 rowspan=1 > C/C++/Python/Java SIMD dynamic programming library for SSE, AVX2 </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Global, Ends-free, Local </td><td colspan=1 rowspan=1 > site </td><td colspan=1 rowspan=1 > J. Daily </td><td colspan=1 rowspan=1 > 2015 </td></tr>
<tr> <td colspan=1 rowspan=1 > Path </td><td colspan=1 rowspan=1 > Smith-Waterman on protein back-translation graph (detects frameshifts at protein level) </td><td colspan=1 rowspan=1 > Protein </td><td colspan=1 rowspan=1 > Local </td><td colspan=1 rowspan=1 > server download </td><td colspan=1 rowspan=1 > M. Gîrdea et al. </td><td colspan=1 rowspan=1 > 2009 </td></tr>
<tr> <td colspan=1 rowspan=1 > PatternHunter </td><td colspan=1 rowspan=1 > Seeded pattern-matching </td><td colspan=1 rowspan=1 > Nucleotide </td><td colspan=1 rowspan=1 > Local </td><td colspan=1 rowspan=1 > download </td><td colspan=1 rowspan=1 > B. Ma et al. </td><td colspan=1 rowspan=1 > 2002–2004 </td></tr>
<tr> <td colspan=1 rowspan=1 > ProbA (also propA) </td><td colspan=1 rowspan=1 > Stochastic partition function sampling via dynamic programming </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Global </td><td colspan=1 rowspan=1 > download </td><td colspan=1 rowspan=1 > U. Mückstein </td><td colspan=1 rowspan=1 > 2002 </td></tr>
<tr> <td colspan=1 rowspan=1 > PyMOL </td><td colspan=1 rowspan=1 > "align" command aligns sequence & applies it to structure </td><td colspan=1 rowspan=1 > Protein </td><td colspan=1 rowspan=1 > Global (by selection) </td><td colspan=1 rowspan=1 > site </td><td colspan=1 rowspan=1 > W. L. DeLano </td><td colspan=1 rowspan=1 > 2007 </td></tr>
<tr> <td colspan=1 rowspan=1 > REPuter </td><td colspan=1 rowspan=1 > suffix tree based </td><td colspan=1 rowspan=1 > Nucleotide </td><td colspan=1 rowspan=1 > Local </td><td colspan=1 rowspan=1 > download </td><td colspan=1 rowspan=1 > S. Kurtz et al. </td><td colspan=1 rowspan=1 > 2001 </td></tr>
<tr> <td colspan=1 rowspan=1 > SABERTOOTH </td><td colspan=1 rowspan=1 > Alignment using predicted Connectivity Profiles </td><td colspan=1 rowspan=1 > Protein </td><td colspan=1 rowspan=1 > Global </td><td colspan=1 rowspan=1 > download on request[permanent dead link] </td><td colspan=1 rowspan=1 > F. Teichert, J. Minning, U. Bastolla, and M. Porto </td><td colspan=1 rowspan=1 > 2009 </td></tr>
<tr> <td colspan=1 rowspan=1 > Satsuma </td><td colspan=1 rowspan=1 > Parallel whole-genome synteny alignments </td><td colspan=1 rowspan=1 > DNA </td><td colspan=1 rowspan=1 > Local </td><td colspan=1 rowspan=1 > download </td><td colspan=1 rowspan=1 > M.G. Grabherr et al. </td><td colspan=1 rowspan=1 > 2010 </td></tr>
<tr> <td colspan=1 rowspan=1 > SEQALN </td><td colspan=1 rowspan=1 > Various dynamic programming </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Local or global </td><td colspan=1 rowspan=1 > server </td><td colspan=1 rowspan=1 > M.S. Waterman and P. Hardy </td><td colspan=1 rowspan=1 > 1996 </td></tr>
<tr> <td colspan=1 rowspan=1 > SIM, GAP, NAP, LAP </td><td colspan=1 rowspan=1 > Local similarity with varying gap treatments </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Local or global </td><td colspan=1 rowspan=1 > server </td><td colspan=1 rowspan=1 > X. Huang and W. Miller </td><td colspan=1 rowspan=1 > 1990-6 </td></tr>
<tr> <td colspan=1 rowspan=1 > SIM </td><td colspan=1 rowspan=1 > Local similarity </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Local </td><td colspan=1 rowspan=1 > servers </td><td colspan=1 rowspan=1 > X. Huang and W. Miller </td><td colspan=1 rowspan=1 > 1991 </td></tr>
<tr> <td colspan=1 rowspan=1 > SPA: Super pairwise alignment </td><td colspan=1 rowspan=1 > Fast pairwise global alignment </td><td colspan=1 rowspan=1 > Nucleotide </td><td colspan=1 rowspan=1 > Global </td><td colspan=1 rowspan=1 > available upon request </td><td colspan=1 rowspan=1 > Shen, Yang, Yao, Hwang </td><td colspan=1 rowspan=1 > 2002 </td></tr>
<tr> <td colspan=1 rowspan=1 > SSEARCH </td><td colspan=1 rowspan=1 > Local (Smith-Waterman) alignment with statistics </td><td colspan=1 rowspan=1 > Protein </td><td colspan=1 rowspan=1 > Local </td><td colspan=1 rowspan=1 > EMBL-EBI FASTA server </td><td colspan=1 rowspan=1 > W. Pearson </td><td colspan=1 rowspan=1 > 1981 (Algorithm) </td></tr>
<tr> <td colspan=1 rowspan=1 > Sequences Studio </td><td colspan=1 rowspan=1 > Java applet demonstrating various algorithms from </td><td colspan=1 rowspan=1 > Generic sequence </td><td colspan=1 rowspan=1 > Local and global </td><td colspan=1 rowspan=1 > code applet </td><td colspan=1 rowspan=1 > A.Meskauskas </td><td colspan=1 rowspan=1 > 1997 (reference book) </td></tr>
<tr> <td colspan=1 rowspan=1 > SWIFT suit </td><td colspan=1 rowspan=1 > Fast Local Alignment Searching </td><td colspan=1 rowspan=1 > DNA </td><td colspan=1 rowspan=1 > Local </td><td colspan=1 rowspan=1 > site </td><td colspan=1 rowspan=1 > K. Rasmussen, W. Gerlach </td><td colspan=1 rowspan=1 > 2005,2008 </td></tr>
<tr> <td colspan=1 rowspan=1 > stretcher </td><td colspan=1 rowspan=1 > Memory-optimized Needleman-Wunsch dynamic programming </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Global </td><td colspan=1 rowspan=1 > Pasteur </td><td colspan=1 rowspan=1 > I. Longden (modified from G. Myers and W. Miller) </td><td colspan=1 rowspan=1 > 1999 </td></tr>
<tr> <td colspan=1 rowspan=1 > tranalign </td><td colspan=1 rowspan=1 > Aligns nucleic acid sequences given a protein alignment </td><td colspan=1 rowspan=1 > Nucleotide </td><td colspan=1 rowspan=1 > NA </td><td colspan=1 rowspan=1 > Pasteur </td><td colspan=1 rowspan=1 > G. Williams (modified from B. Pearson) </td><td colspan=1 rowspan=1 > 2002 </td></tr>
<tr> <td colspan=1 rowspan=1 > UGENE </td><td colspan=1 rowspan=1 > Opensource Smith-Waterman for SSE/CUDA, Suffix array based repeats finder & dotplot </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > UGENE site </td><td colspan=1 rowspan=1 > UniPro </td><td colspan=1 rowspan=1 > 2010 </td></tr>
<tr> <td colspan=1 rowspan=1 > water </td><td colspan=1 rowspan=1 > Smith-Waterman dynamic programming </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > Local </td><td colspan=1 rowspan=1 > EMBL-EBIPasteur </td><td colspan=1 rowspan=1 > A. Bleasby </td><td colspan=1 rowspan=1 > 1999 </td></tr>
<tr> <td colspan=1 rowspan=1 > wordmatch </td><td colspan=1 rowspan=1 > k-tuple pairwise match </td><td colspan=1 rowspan=1 > Both </td><td colspan=1 rowspan=1 > NA </td><td colspan=1 rowspan=1 > Pasteur </td><td colspan=1 rowspan=1 > I. Longden </td><td colspan=1 rowspan=1 > 1998 </td></tr>
<tr> <td class="highlighted"  colspan=1 rowspan=1 > YASS </td><td colspan=1 rowspan=1 > Seeded pattern-matching </td><td class="highlighted"  colspan=1 rowspan=1 > Nucleotide </td><td colspan=1 rowspan=1 > Local </td><td colspan=1 rowspan=1 > server download </td><td colspan=1 rowspan=1 > L. Noe and G. Kucherov </td><td colspan=1 rowspan=1 > 2004 </td></tr>
</table> <br> <h3>Sentence(s)</h3>1. YASS is a pairwise sequence alignment software for nucleotide sequences. <br> <br><br><b>Categories:</b><br><b>Numerical:</b> Uses math skills for writing the sentence. Uses math operations (addition, subtraction, average, etc.), counting numbers, frequency of items, etc.<br> <b>Commonsense:</b> Uses commonsene knowledge about situations humans encounter in everyday life – physical objects, word meanings, social or physical domains, peoples' intentions, etc.<br> <b>Temporal:</b> When concepts related to time and temporal aspects are required for writing the sentence. Duration, ordering, typical time of events, time differences between events.<br> <b>Table:</b> Writing the sentence requires knowledge from the table except: the highlighted cells, row and column headers, table titles, section titles, section texts.<br> <b>Out of Table:</b> Writing the sentence requires external knowledge that is not present in the table nor common sense. Knowledge about some specific domain comes under this category. </body></html>
